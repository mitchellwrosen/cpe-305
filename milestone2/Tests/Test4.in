2000 2 .01 10 1000
GCa 0 0 0 1 1
