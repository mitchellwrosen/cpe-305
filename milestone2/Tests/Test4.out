<Movie frameRate="24">
<Objects>
<Point id="GCa[-1,0,0]" color="FFFFFF"/>
<Point id="GCa[0,-1,0]" color="FFFFFF"/>
<Point id="GCa[0,0,-1]" color="FFFFFF"/>
<Point id="GCa[0,0,0]" color="F0F0F"/>
<Point id="GCa[0,0,1]" color="FFFFFF"/>
<Point id="GCa[0,1,0]" color="FFFFFF"/>
<Point id="GCa[1,0,0]" color="FFFFFF"/>
</Objects>
Starting simulation with 7 particles and 21 forces
Update force between GCa[-1,0,0] and GCa[0,-1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
<Frame time="0.000">
<Update id="GCa[-1,0,0]" position="-1.000, 0.000, 0.000"/>
<Update id="GCa[0,-1,0]" position="0.000, -1.000, 0.000"/>
<Update id="GCa[0,0,-1]" position="0.000, 0.000, -1.000"/>
<Update id="GCa[0,0,0]" position="0.000, 0.000, 0.000"/>
<Update id="GCa[0,0,1]" position="0.000, 0.000, 1.000"/>
<Update id="GCa[0,1,0]" position="0.000, 1.000, 0.000"/>
<Update id="GCa[1,0,0]" position="1.000, 0.000, 0.000"/>
</Frame>
Update force between GCa[-1,0,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 2.000000 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,1,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 2.000000 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.000000 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 2.000000 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414214 wait: 1000.000000
Part GCa[0,0,-1] moves at (0.000000000, 0.000000000, 0.000000007) to (0.000000000, 0.000000000, -0.999996505) at 1000.000000000)
Part GCa[0,1,0] moves at (0.000000000, -0.000000007, 0.000000000) to (0.000000000, 0.999996505, 0.000000000) at 1000.000000000)
Update force between GCa[0,0,-1] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Part GCa[1,0,0] moves at (-0.000000007, 0.000000000, 0.000000000) to (0.999996505, 0.000000000, 0.000000000) at 1000.000000000)
Update force between GCa[0,0,-1] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Part GCa[0,0,1] moves at (0.000000000, 0.000000000, -0.000000007) to (0.000000000, 0.000000000, 0.999996505) at 1000.000000000)
Update force between GCa[0,0,1] and GCa[1,0,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Part GCa[0,0,0] moves at (0.000000000, 0.000000000, 0.000000000) to (0.000000000, 0.000000000, 0.000000000) at 1000.000000000)
Update force between GCa[0,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Part GCa[-1,0,0] moves at (0.000000007, 0.000000000, 0.000000000) to (-0.999996505, 0.000000000, 0.000000000) at 1000.000000000)
Update force between GCa[-1,0,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Part GCa[0,-1,0] moves at (0.000000000, 0.000000007, 0.000000000) to (0.000000000, -0.999996505, 0.000000000) at 1000.000000000)
Update force between GCa[-1,0,0] and GCa[0,-1,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999993 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[0,1,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Update force between GCa[0,1,0] and GCa[1,0,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414209 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999993 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999993 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999997 wait: 1000.000000
Part GCa[-1,0,0] moves at (0.000000014, 0.000000000, 0.000000000) to (-0.999986018, 0.000000000, 0.000000000) at 2000.000000000)
Part GCa[1,0,0] moves at (-0.000000014, 0.000000000, 0.000000000) to (0.999986018, 0.000000000, 0.000000000) at 2000.000000000)
Update force between GCa[-1,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999972 wait: 1000.000000
Part GCa[0,0,-1] moves at (0.000000000, 0.000000000, 0.000000014) to (0.000000000, 0.000000000, -0.999986018) at 2000.000000000)
Part GCa[0,0,0] moves at (0.000000000, 0.000000000, 0.000000000) to (0.000000000, 0.000000000, 0.000000000) at 2000.000000000)
Update force between GCa[0,0,-1] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Part GCa[0,-1,0] moves at (0.000000000, 0.000000014, 0.000000000) to (0.000000000, -0.999986018, 0.000000000) at 2000.000000000)
Part GCa[0,0,1] moves at (0.000000000, 0.000000000, -0.000000014) to (0.000000000, 0.000000000, 0.999986018) at 2000.000000000)
Update force between GCa[0,-1,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,-1,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Part GCa[0,1,0] moves at (0.000000000, -0.000000014, 0.000000000) to (0.000000000, 0.999986018, 0.000000000) at 2000.000000000)
Update force between GCa[0,0,-1] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
<Frame time="2000.000">
<Update id="GCa[-1,0,0]" position="-1.000, 0.000, 0.000"/>
<Update id="GCa[0,-1,0]" position="0.000, -1.000, 0.000"/>
<Update id="GCa[0,0,-1]" position="0.000, 0.000, -1.000"/>
<Update id="GCa[0,0,0]" position="0.000, 0.000, 0.000"/>
<Update id="GCa[0,0,1]" position="0.000, 0.000, 1.000"/>
<Update id="GCa[0,1,0]" position="0.000, 1.000, 0.000"/>
<Update id="GCa[1,0,0]" position="1.000, 0.000, 0.000"/>
</Frame>
Update force between GCa[-1,0,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[1,0,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[0,1,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999972 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999972 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999986 wait: 1000.000000
Update force between GCa[0,1,0] and GCa[1,0,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414194 wait: 1000.000000
Part GCa[-1,0,0] moves at (0.000000021, 0.000000000, 0.000000000) to (-0.999968541, 0.000000000, 0.000000000) at 3000.000000000)
Part GCa[0,0,0] moves at (0.000000000, 0.000000000, 0.000000000) to (0.000000000, 0.000000000, 0.000000000) at 3000.000000000)
Update force between GCa[-1,0,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Part GCa[0,1,0] moves at (0.000000000, -0.000000021, 0.000000000) to (0.000000000, 0.999968541, 0.000000000) at 3000.000000000)
Part GCa[1,0,0] moves at (-0.000000021, 0.000000000, 0.000000000) to (0.999968541, 0.000000000, 0.000000000) at 3000.000000000)
Update force between GCa[0,1,0] and GCa[1,0,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Part GCa[0,0,1] moves at (0.000000000, 0.000000000, -0.000000021) to (0.000000000, 0.000000000, 0.999968541) at 3000.000000000)
Update force between GCa[0,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Part GCa[0,-1,0] moves at (0.000000000, 0.000000021, 0.000000000) to (0.000000000, -0.999968541, 0.000000000) at 3000.000000000)
Update force between GCa[0,-1,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999937 wait: 1000.000000
Part GCa[0,0,-1] moves at (0.000000000, 0.000000000, 0.000000021) to (0.000000000, 0.000000000, -0.999968541) at 3000.000000000)
Update force between GCa[0,0,-1] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,-1,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[1,0,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,0,1] and GCa[0,1,0]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999937 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,0,1]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.999937 wait: 1000.000000
Update force between GCa[-1,0,0] and GCa[0,1,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,-1,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414169 wait: 1000.000000
Update force between GCa[0,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999969 wait: 1000.000000
Part GCa[0,-1,0] moves at (0.000000000, 0.000000028, 0.000000000) to (0.000000000, -0.999944073, 0.000000000) at 4000.000000000)
Part GCa[0,0,-1] moves at (0.000000000, 0.000000000, 0.000000028) to (0.000000000, 0.000000000, -0.999944073) at 4000.000000000)
Update force between GCa[0,-1,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414134 wait: 1000.000000
Part GCa[0,0,0] moves at (-0.000000000, -0.000000000, -0.000000000) to (-0.000000000, -0.000000000, -0.000000000) at 4000.000000000)
Part GCa[1,0,0] moves at (-0.000000028, 0.000000000, 0.000000000) to (0.999944073, 0.000000000, 0.000000000) at 4000.000000000)
Update force between GCa[0,0,0] and GCa[1,0,0]. (0.000000000, 0.000000000, 0.000000000)->(0.000000000, 0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999944 wait: 1000.000000
Part GCa[-1,0,0] moves at (0.000000028, 0.000000000, 0.000000000) to (-0.999944073, 0.000000000, 0.000000000) at 4000.000000000)
Update force between GCa[-1,0,0] and GCa[0,0,-1]. (0.000000000, 0.000000000, -0.000000000)->(0.000000000, 0.000000000, -0.000000000)

Deltaspeed 0.000000 dist 1.414134 wait: 1000.000000
Update force between GCa[0,0,-1] and GCa[0,0,0]. (0.000000000, 0.000000000, 0.000000000)->(-0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 0.999944 wait: 1000.000000
Part GCa[0,1,0] moves at (0.000000000, -0.000000028, 0.000000000) to (0.000000000, 0.999944073, 0.000000000) at 4000.000000000)
Update force between GCa[0,1,0] and GCa[1,0,0]. (0.000000000, -0.000000000, 0.000000000)->(0.000000000, -0.000000000, 0.000000000)

Deltaspeed 0.000000 dist 1.414134 wait: 1000.000000
</Movie>